web/packages/demo/www/index.js

```diff
@@ -69,26 +71,48 @@ async function loadFile(file) {
         return;
     }
     hideSample();
+    sampleFileInput.selectedIndex = 1;
+    previousSelectedIndex = 1;
+    localFileLabel.innerHTML = file.name;
     const data = await new Response(file).arrayBuffer();
     load({ data, ...config });
 }
 
 function loadSample() {
     const swfData = sampleFileInput[sampleFileInput.selectedIndex].swfData;
     if (swfData) {
+        localFileLabel.innerHTML = "";
         showSample(swfData);
         load({ url: swfData.location, ...config });
     } else {
-        hideSample();
-        unload();
+        if (sampleFileInput[sampleFileInput.selectedIndex].value === "local") {
+            localFileInput.click();
+        } else {
+            localFileLabel.innerHTML = "";
+            hideSample();
```

Comment:
Cannot comment on `hideSample` itself, so commenting here.
It can be reduced to just:
```js
function hideSample() {
    authorContainer.classList.add("hidden");
}
```